packages/svelte/src/lib/components/primitives/AmplifyButton.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	export let type: 'submit' | 'button' = 'button';
+	export let fullWidth: boolean | string = false;
+	export let size: 'small' | 'medium' | 'large' = 'medium';
+	export let variation: 'primary' | 'default' | 'link' = 'default';
+	export let fontWeight: 'normal' | 'bold' | 'lighter' = 'normal';
+
+	import { createEventDispatcher } from 'svelte';
```

Comment:
Style nit -- should this be at the top of this script? Not familiar with svelte coding conventions.

Reply:
I can move to the top no prob